source/game/kart/KartObjectProxy.hh

```diff
@@ -113,19 +113,22 @@ public:
     u16 tireCount() const;
     bool hasFloorCollision(const WheelPhysics *wheelPhysics) const;
 
-    Abstract::List *list() const;
+    Abstract::List<KartObjectProxy> list() const;
 
 protected:
     void apply(size_t idx);
 
     const KartAccessor *m_accessor;
 
 private:
+    void InitList();
     static void ApplyAll(const KartAccessor *pointers);
 
     // Used to initialize multiple KartObjectProxy instances at once
     // Lists are created on the stack in KartObject::Create
-    static Abstract::List *s_list;
+    Abstract::Link m_link;
+    static bool s_listInitialized;
```

Comment:
The same thing applies with this list. This should be a static list, but we see in 8058f5d4 that there's a clear starting point for when the list is initialized.